### PR TITLE
Refs 106: Set NODE_BUILD_VERSION from .nvmrc in pr_check

### PR DESCRIPTION
**Pull request by mshriver**
15 was from the template, didn't catch .nvmrc has 16 as the major version.

Set it dynamically so we don't have to update it in multiple places.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-106

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: .rhcicd/pr_check.sh [hunk 1]
```diff
@@ -9,7 +9,16 @@ export COMPONENT_NAME="content-sources-backend"  # name of resourceTemplate comp
 export IMAGE="quay.io/cloudservices/content-sources-frontend"
 export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
 export APP_ROOT=$(pwd)
-export NODE_BUILD_VERSION=15
+
+# set NODE_BUILD_VERSION based on nvmrc or default to 16
+if test -f "${APP_ROOT}/.nvmrc"; then
+    # -P for perl, look-behind for v, match the major version number, look ahead for .
+    # -o output only matched major version number, don't output (?=) groups
+    export NODE_BUILD_VERSION=$(grep -Po '(?<=v)[0-9]+(?=\.)' .nvmrc)
```
Comment (Andrewgdewar) [inline .rhcicd/pr_check.sh:17]: very nice

**Review by Andrewgdewar — APPROVED** [review]
LGTM, much version! very Ack!

**Pull request merged**